src/index.html

```diff
@@ -7,8 +7,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo"><img src="./images/logo.png" alt="MOYO logo"></a>
+      <nav class="nav_bar">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="link is-active" href="#">Apple</a></li>
+          <li class="nav__item"><a class="link" href="#">Samsung</a></li>
+          <li class="nav__item"><a class="link" href="#">Smartphones</a></li>
+          <li class="nav__item">
+            <a class="link "
+               data-qa="hover"
+               href="#">Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a 
              class="link "
              data-qa="hover"
              href="#"
            >
              Laptops & Computers
            </a>
```
Checklist:
Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/3b95db60-b40a-4c89-b919-d71d2d54a595)